actions/TestTaker.php

```diff
@@ -230,16 +230,6 @@ public function editSubject()
         $this->setView('form_subjects.tpl');
     }
 
-    /**
```

Comment:
For those deleted methods, are we sure we are not removing specific rules for specific resources that were set before?